Marlin/ultralcd.cpp

```diff
@@ -805,13 +807,29 @@ static void lcd_prepare_menu() {
 
 #endif // DELTA_CALIBRATION_MENU
 
-inline void line_to_current(AxisEnum axis) {
-  #if ENABLED(DELTA)
-    calculate_delta(current_position);
-    plan_buffer_line(delta[X_AXIS], delta[Y_AXIS], delta[Z_AXIS], current_position[E_AXIS], manual_feedrate[axis]/60, active_extruder);
-  #else
-    plan_buffer_line(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], manual_feedrate[axis]/60, active_extruder);
-  #endif
+/**
+ * If the manual move hasn't been fed to the planner yet,
+ * and the planner can accept one, send immediately
+ */
+inline void manage_manual_move() {
+  if (manual_move_axis != (int8_t)NO_AXIS && !planner_is_full()) {
+    #if ENABLED(DELTA)
```

Comment:
No, sorry i have not been precise in what i said.
With planner_is_full() you have to wait up to 16 moves until the machine stops - with (movesplanned() <= 3) only 3.
3 because 1 or 2 does not give the planer the chance to optimize the accelerations - you'l have stops between the single moves.

Let's begin wit turning the knob.
The firs enqueued moves are all short - speed is low - until the buffer is full (or we reach an other limit (3)).
From there on the steps size will increase. 
Finally the steps are long enough (take enough time) to turn the encoder back and forth without enqueuing new moves - only changing the size of the next move.
Finally you are able to send moves with a complete axis length.
Waiting for 16 full axis length moves, until the tool stops, is a bit long? Isn't it?